src/utils/azure_storage.ts

```diff
@@ -263,15 +263,17 @@ export const getBlobAsObject = async <A, O, I>(
 
 // TABLE STORAGE FUNCTIONS AND TYPES
 
-// Basic type for a table entity
+/**
+ * Legacy. Basic type for a table entity for the library `azure-storage`.
+ */
 /* eslint-disable @typescript-eslint/naming-convention */
-export const TableEntity = t.interface({
+export const TableEntityAzureStorage = t.interface({
```

Comment:
since we are changing an already exported and well-known type `TableEntity`, this type of change will introduce a breaking change at runtime for old adopters. I would instead leave the name as it is and create a new type called for example `TableEntityAzureDataTables` inside `utils/data_tables.ts` for new adopters, eliminating a breaking change. do you agree?

Reply:
You're right. I'll change it right away.

Reply:
Resolved in [491818b](https://github.com/pagopa/io-functions-commons/pull/384/commits/491818b5019eaf91be790915b87b44909b57785e)